README.md

```diff
@@ -8,6 +8,7 @@
 ## Project setup
 * Download the [.env](https://start.1password.com/open/i?a=7BICDIKH2ZHQZIH6N3APRMZKLU&v=zu4fcddpxze65mjtzpq6fcadim&i=4qxmcvqnezauxblqwo7hpzzlnu&h=team-swec.1password.ca) and put it in the project folder
 * Run ```make get```
+  * If there is an issue with installing Pinia, try using npm 7 (```nvm use 15```)
```

Comment:
Just want to double check this is expected? Says npm 7 and then 15

Reply:
Yes, node v15 corresponds with npm v7.